server/setup.py

```diff
@@ -45,7 +45,7 @@
         'ftfy',
         'torch',
         'regex',
-        'torchvision',
+        'torchvision<=0.13.0' if sys.version_info <= (3, 7, 2) else 'torchvision',
```

Comment:
Please manually check whether this change works as expected.

Reply:
<img width="584" alt="Screen Shot 2022-11-25 at 3 42 53 PM" src="https://user-images.githubusercontent.com/16580382/203927518-62cce6e3-c955-46d4-9a9c-f5824bbe675f.png">
<img width="587" alt="Screen Shot 2022-11-25 at 3 42 41 PM" src="https://user-images.githubusercontent.com/16580382/203927531-2896ac7c-56fa-4edd-9911-3bdb3f13c4d9.png">

Reply:
Different python version: `3.8.15` and `3.7.0`, we have different `torchvision` version